### DISCUSSION TITLE
✨ Synapse

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2190] [Requests]
### Description

Synapse is an open-source [Matrix](https://matrix.org/) homeserver written and maintained by the Matrix.org Foundation. We began rapid development in 2014, reaching v1.0.0 in 2019. Development on Synapse and the Matrix protocol itself continues in earnest today.

Briefly, Matrix is an open standard for communications on the internet, supporting federation, encryption and VoIP. Matrix.org has more to say about the [goals of the Matrix project](https://matrix.org/docs/guides/introduction), and the [formal specification](https://spec.matrix.org/) describes the technical details.

### Documentation link

https://element-hq.github.io/synapse/latest/

### Last application release & date

1.99.0 (16/01/2024)

### Application license

AGPL-3.0 license

### Source code repository link

https://github.com/element-hq/synapse

### Docker image link

https://github.com/element-hq/synapse/pkgs/container/synapse

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @AAGaming00

**Reply by AAGaming00** [reply]
~~Now that it's further along, you may want to look into Conduit as well.~~ Oh wow nevermind you already have conduit.

Also, happy to see this project is still going after this long! 🎉

**Comment by OriginalSimon** [comment]
Synapse is available in this [custom store](https://github.com/sergi0g/runtipi-appstore
).
